h2/events.py

```diff
@@ -334,9 +334,18 @@ def __init__(self):
         #: peer and so can safely be resent.
         self.last_stream_id = None
 
+        #: Additional debug data that can be appended to GOAWAY frame.
+        self.additional_data = None
+
     def __repr__(self):
-        return "<ConnectionTerminated error_code:%s, last_stream_id:%s>" % (
-            self.error_code, self.last_stream_id
+        return (
+            "<ConnectionTerminated error_code:%s, last_stream_id:%s, "
+            "additional_data:%s>" % (
+                self.error_code,
+                self.last_stream_id,
+                (_bytes_representation(self.additional_data[:20])
+                 if self.additional_data else None)
```

Comment:
This has gotten just a little bit confusing. It might be better, and more robust, if we drag the `if None` check into `_bytes_representation`, such that calling `_bytes_representation(None)` returns `None` (which is about what you'd expect anyway).